client/src/components/Kanban/CardForm/InnerForm/InnerForm.tsx

```diff
@@ -9,12 +9,14 @@ type InnerFormProps = {
   formAction: Dispatch<SetStateAction<boolean>>;
 };
 
+// IMPORTANT TODO - Change this to a formik form instead (see login/signup forms) to keep consistent cross app
```

Comment:
@street-jackal

Reply:
The CardForm/InnerForm stuff was @moffatethan , so I'll tag him here.